core/src/query.rs

```diff
@@ -13,47 +17,92 @@ pub struct MongoQuery {
 }
 
 impl MongoQuery {
-    // Create a new MongoStatement with StmtKind::Query on the connection currentDB.
```

Comment:
This old comment seemed out of date/inaccurate to how we are doing this now so I updated it!